### DISCUSSION TITLE
Welcome to Multi-Camera-Person-Re-Identification Discussions!

### DISCUSSION THREAD

**Discussion by SurajDonthi** [SurajDonthi/Multi-Camera-Person-Re-Identification#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lalithbharadwaj** [comment]
the bounding_box_train, bounding_box_test,  query files should contain XML files of the labelled coordinates by labelImg?

**Reply by SurajDonthi** [reply]
They should contain the images after passing through a Bounding Box detection of people. So each of these folders contains cropped persons in the image.

**Reply by lalithbharadwaj** [reply]
thankyou, also may I know how these are differentiated as junk, good and distractor (MatLab files) like we have to make our own dataset how will we make them?

**Reply by SurajDonthi** [reply]
@lalithbharadwaj can you please elaborate on what you mean by "junk", "good" and "distractor". Please provide more context for what you mean by MATLAB files in this scenario.

If you want to create a dataset from scratch, you'll need to  have the CCTV recordings of the people walking between different Cameras. From this you need to extract the people and the respective timestamp. You can create a query & database sets.

You can use the database to create a temporal distribution & also train the CNN model for the same person.

**Reply by lalithbharadwaj** [reply]
yes, I have the idea of creating the datasets, but when i analysed the market-1510 there were 5 files 

1) bounding box train
2) bounding box test
3) query
4) gt bbox
5) gt query

gt query folder had MATLAB files where the query file containing images are bifurcated into junk, good and distractor.

I wanted to know how to create those MatLab files so that will be able to do that.

![Capture](https://user-images.githubusercontent.com/95376140/156871264-88996c3b-313b-4838-9c9a-7ab948582757.PNG)
![Capture1](https://user-images.githubusercontent.com/95376140/156871266-93ee14e5-6c2c-4fa4-a742-41d69265d5b5.PNG)
